CARDS-1799: LFS mode: Timeline unresponsive

veronikaslc opened the pull request:
Fixed leftover moment to luxon migration artifacts

Commit:
CARDS-1799: LFS mode: Timeline unresponsive
Fixed leftover moment to luxon migration artifacts

sashaandjic commented:
The chart entries appear on one side only. Here is the comparison of the look of the old timeline and new one in this PR:
![image](https://user-images.githubusercontent.com/23243244/173420350-ab17784c-ae3e-4700-b097-7af2debb7a82.png)

marta- commented:
> The chart entries appear on one side only. Here is the comparison of the look of the old timeline and new one in this PR (same dates were used for both timelines)

This issue is related to the mui5 upgrade (CARDS-1441) so it's slightly out of scope. Let's report it as a separate task.

Commit:
CARDS-1799: LFS mode: Timeline unresponsive
Round time intervals to the nearest integer

Commit:
CARDS-1441: Upgrade material ui and mui/lab to version 5+
NumberFormatCustom component should forward the ref prop instead of the inputRef prop.

Commit:
CARDS-1441: Upgrade material ui and mui/lab to version 5+
Updated the Timeline component with new prop to display the events on alternating sides.

sashaandjic reviewed (approved):
tested - both problems are now resolved
![image](https://user-images.githubusercontent.com/23243244/173616721-6a16c154-4614-4e55-aee7-e13f963d3e60.png)
